README.md

```diff
@@ -1,18 +1,39 @@
 # mlr3book
 
-[![Travis build status](https://travis-ci.org/mlr-org/mlr3book.svg?branch=master)](https://travis-ci.org/mlr-org/mlr3book)
+[![Build Status](https://img.shields.io/travis/mlr-org/mlr3book/master?label=Linux&logo=travis&style=flat-square)](https://travis-ci.org/mlr-org/mlr3book)
 [![StackOverflow](https://img.shields.io/badge/stackoverflow-mlr3-orange.svg)](https://stackoverflow.com/questions/tagged/mlr3)
 
 Package to build the [mlr3](https://mlr3.mlr-org.com) [bookdown](https://bookdown.org/) book.
-The rendered book can be found [here](https://mlr3book.mlr-org.com).
 
-To install all necessary dependencies for the book, install the this R package using [remotes](https://cran.r-project.org/package=remotes):
+## Rendered Versions
+
+- [HTML](https://mlr3book.mlr-org.com)
+
+- [PDF](https://mlr3book.mlr-org.com/mlr3book.pdf)
+
+## Building the book
+
+To install all necessary dependencies for the book, install this R package using [remotes](https://cran.r-project.org/package=remotes):
 
 ```r
 remotes::install_github("mlr-org/mlr3book", dependencies = TRUE)
 ```
 
-To build the HTML version of the book, run the following R command in the repository root:
+To build the book, run one of the following commands:
+
+```r
+# HTML
+withr::with_dir("bookdown", bookdown::render_book("index.Rmd", 
```

Comment:
This is now the 3rd way to build the book (this, Makefile, helper function in the mlr3book package).

Reply:
Oh yes, this and the `pdf_book` call could be replaced by only `mlr3book::render_mlr3book()`. 👍